provider/config.go

```diff
@@ -72,13 +72,6 @@ func DecodeConfig(b []byte) (*Config, error) {
 		return config, errors.New("Config error: must declare a service name prefix")
 	}
 
-	// Aiven only allow 16 characters for the service name. The instanceID from Cloud Foundry
-	// is hashed with crc32, then represented as 8 hexadecimal digits: this gives us 8
-	// characters.
-	if len(config.ServiceNamePrefix) > 8 {
-		return config, errors.New("Config error: service name prefix cannot be longer than 8 characters")
-	}
```

Comment:
A thought, we probably still should have a check here, just with a higher threshold. I think the new service name limit is 64 chars, so we should constrain this to leave enough space for the GUID to be added.

Reply:
Done